docker/reqmgr2ms-unmerged/install.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
We found that bash may have vulnerabilities and if you don't really use bash specific features it is better to switch to simpler /bin/sh.

Reply:
Based on my last night experience losing hours to debug shell versions - I am hardly inclined not to touch this one right now.